src/content/advanced/cost optimization/index.md

```diff
@@ -0,0 +1,49 @@
+---
+linkTitle: Cost-optimization
+title: Cost optimization tools
+description: Cost is always a concern when running IT infrasturcture. This document is a curated list of cost optimization tools that we have found to be valuable for our customers and oursleves.
+weight: 100
+menu:
+  main:
+    parent: advanced
+user_questions:
+- What cost optimization tools do you reccomend?
+- How can I tune my setup to optimize for cost?
+aliases:
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+---
+
+# Cost optimization in Kubernetes
+
+Adoption of Kuberenetes still requires monitoring cost drivers. Improving resource utilization allows you to get the most out of your infrastructure, while maintaining your budget. This document is comrpised of a curated list of helpful tools on the journey to cost optimization.
+
+## Visualization
+
+- Open Core Cost visualization tool https://kubecost.com
```

Comment:
I think instead of "open core" it should be "open source"

Reply:
@pipo02mix can you verify which is the correct one?

Reply:
Is not really open source. It has a part which it is but if you want to use the product for commercial purposes you have to pay

Reply:
So what is `Open Core` then?

Reply:
They open-sourced part of the product but to get the whole experience you have to pay